### ISSUE TITLE
Add  "Cash by mail" as a payment option

### ISSUE THREAD

**Issue by mutagenfork** [bisq-network/bisq#1101]
Bisq advertises privacy as one of the features it offers; to whit:

"Private – no one except trading partners exchange personally identifying data. All personal data is stored locally."

However, this is not true.  All of the currently listed payment methods require signing up with a bank or financial institution and sharing with them rather invasive details about one's identity and financial history.  

The currently allowed payment methods are also unavailable to anyone who is unbanked, such as an immigrant, or a resident of a country where the banking system is underdeveloped and only available to the wealthy. 

Therefore, I'd like to request that bisq add support for payments by "cash in person" and "cash by mail". 

Note that this is already an option at several other exchanges/companies:

https://paxful.com/buy-bitcoin/cash-by-mail
https://localbitcoins.com/buy-bitcoins-online/cash-by-mail/
https://moneroforcash.com/

**Comment by mutagenfork** [comment]
Here's how "cash by mail" payments for bitcoin (BTC) are handled on other sites:

* Bitcoin buyer packages cash in a [tamper-resistant cash control bag](http://store.ampaconline.com/55X85-Cash-Control-Bag-White-Secure-Closure-P99.aspx)

* Buyer films packaging with the tracking number already affixed to packaging. 

* Buy sends the package with USPS tracking.  

* Seller films the opening of the package, making sure that the tracking number provided by the sender is visible in the video.  

* Seller counts the cash, and [authenticates the cash](
https://www.youtube.com/watch?v=Ybt04VtOmzE).

* If the unpackaging video shows there is nothing in the package, the cash is short,  or the cash is counterfeit then then the seller can lodge a dispute.

* If the cash is short, then the arbitrator can ask the buyer to send more cash to the seller, or refund some of the BTC.

* If there is extra cash, the the arbitrator can ask the seller to send the buyer extra BTC as a separate transaction.

* If the dispute goes to adjudication:

*  For the buyer to win, they'd do so on the basis that the unpackaging video showed some evidence of either tampering with the tracking sticker or the package.  Or the seller didn't do a proper job of filming the unpackaging. 

* For seller to win, they would need to somehow prove that they did not receive the package or it did not contain the contents promised. This is a tall order. Adjudication would be able to see that the tracking number provided showed that the package was delivered (and possibly signed for).  The packaging video would also clearly show that the right amount of cash going into the package.  In order to succeed with a scam, the seller would need to tamper with the tracking sticker and transfer it to another package and video that as the fake video they would submit to adjudicators--in practice, it's quite a bit of work to make the packaging look similar enough to fool the arbitrators.   

* On LocalBitcoins the adjudication standard is that if the cash doesn't make it to the Bitcoin seller for any reason, the default resolution (with rare exceptions) is that the escrow will be resolved in the favor of the seller.

**Comment by dan-da** [comment]
+1

nice writeup of cash by mail procedures.   Maybe you can do one for cash in person also?  perhaps should be a separate issue.

**Comment by cbeams** [comment]
We'll do a short call about 'cash via mail' ideas later today. Anyone is free to join, and @mutagenfork it would be great to have you there!

https://www.google.com/calendar/event?eid=N2RhM3BiZ3BnYTJ1OTk5MDBoNHZkOGQxcDggYWRmaXVpb2txOWp0YWM4dDlrZW9tYzAwYjBAZw

**Comment by ManfredKarrer** [comment]
I am not sure sure if I can make it (release), but will try.

**Comment by cbeams** [comment]
Updated zoom call link: https://zoom.us/j/301252266

**Comment by cbeams** [comment]
@mutagenfork, we're here now: https://zoom.us/j/301252266

**Comment by dscotese** [comment]
I'm glad this issue is still open.  I left a comment in #1110 about arbitration.  I think that arbitration is wonderful even if it fails sometimes.  The trick is to positively identify an arbiter (sign stuff with the private key of an address from which you send some sort of bond or something?) so that their reputation can follow them.  I also would like to see a cash by mail option in BISQ.

**Comment by ripcurlx** [comment]
@alfsbs Do you think this payment method is something we should consider?

**Comment by aejontargaryen** [comment]
bitcoin does have great benefit to the unbanked; and vice versa
i bought my first bitcoin through paxful. 
i know that gift cards starts going a bit 'out there' but could it work to start using only certain prepaid debit cards? not charging the card but, exchanging its ownership code..? 
The back and forth style exchange process of verifying card number etc is very similar to the bisq trade protocol.
did not mean to butt-in . just saw similarities.

**Comment by dscotese** [comment]
Demand for cash by mail might go up quite a bit because localbitcoins just announced that they will be disabling all cash by mail ads on Dec. 27th.  There is an effort to stop them.

**Comment by dmp1ce** [comment]
Another exchange, Paxful, is adding KYC steps in order to trade local and by mail cash trades. Even if Bisq only allowed cash trades as a way to exchange contact information securely, I think it would be a valuable feature.

Currently, the way local cash trades is implemented, I only use Bisq to share contact information securely because it is too much of a hassle to complete local trades which require a laptop.

I think this feature should be revisited as more and more centralized exchanges are regulated.

**Comment by dan-da** [comment]
+1 for in-person trades to be contact info only.   Or at least have it as an option.  Basically it would work as present, but without the multi-sig escrow.   Then people might actually use it....

**Comment by dave2014btc** [comment]
Hello, i am trader on localbitcoins and paxful. done just over 1000 cash by mail trade, had about 5 disputed in the whole time. thats about 0.5% of trades. i think this method of payment needs to be added. the only reason i do not use bisq is i only trade with cash in person or mail and they is not enough incentive to use bisq yet with my methods. once it accepts cash by mail i would more than likely direct customers from other websites as i like the idea of a decentralised exchange.  im just surprised its not already implemented as less risk than in person.

**Comment by crocket** [comment]
@dave2014btc Can you teach us how to do mail-by-cash tranasactions anonymously? I'm trying to use mail-by-cash deals on localmonero. But, I have never done cash deals with other people.

I imagine that I could do so pseudonomously if I bought and installed password-protected delivery storage in my apartment where many people live. When I buy cash with XMR, I would just have cash delivered to the storage box which is shared with my neighbors.

If I have cash delivered to my home address, my opponent learns my home address. I imagine that in the near future, governments may set up cash-by-mail traps on localmonero to punish random suckers who want to buy XMR without leaving traces in the system.

**Comment by dave2014btc** [comment]
I do not try to do it anonymously , but I use po box to protect myself from
bad actors. If someone really wanted they could buy xmr from you and send
package with a tracker hidden inside the cardboard on the box and follow it
exactly to you. But your idea sounds decent but wouldn't be surprised if
someone ripped it off the wall knowing they was cash in it. Best way to use
po box costs a bit but saves you from most bad actors. Btw I'm always
buying BTC and crypto for cash in mail.

**Comment by crocket** [comment]
@dave2014btc A tracker is unlikely to be used but is still physically possible. How is post office box any different except that it's more difficult to rip off such a box and there are CCTVs around it?

Why do you always buy cryptocurrencies by cash in mail unless you are trying to buy cryptocurrencies privately?

Do you have any better idea for private transactions?

**Comment by dave2014btc** [comment]
The po box at the post office is not anonymous , also it's not a physically
secure box just a box in a room behind the counter. They are cameras if
someone was to take something but I always ask people to send special
delivery as you can insure cash in the u.k. but it means I don't have to
give my address to random people and it also means that I can collect my
mail at 8am even tho it won't be delivered till 1pm also untracked mail
seems to arrive a day faster. They are pros and cons. Even police or
official people will need a warrant of some sort to get my information from
the post office. I have other ideas on how you could do it but just adds
more risk on money being lost.

**Comment by crocket** [comment]
@dave2014btc I want to hear your ideas. Perhaps, we can develop this idea somewhere else.

I specifically don't want governments and corporations to be able to trace my transactions. I also don't want whom I make deals with to know my address. I want privacy.

By the way, you can detect trackers by using various EMF meters. If it emits wireless signal, EMF meters can detect electromagnetic frequencies. Trackers have to emit strong wireless signals if they want to communicate with nearby cell towers.

**Comment by ripcurlx** [comment]
Reopening as there is still an ongoing discussion.

**Comment by dscotese** [comment]
I have a customer who uses the address of Gun Owners of America as the return address on what he sends me.  He prefers to forfeit his money rather than anyone knowing his actual address.  I am unable to send him information except through the encrypted channels he has chosen to use.  He found me because of a website, not a DEX like Bisq.  I'd prefer my reputation to be maintained in a decentralized way.  It is, to some extent, but it is "centered" on a few websites.  I have an account on keybase too (so it's one of them), and I like the network their "follow" feature creates.

Anyway, what is needed is the set of questions a Cash By Mail (CBM) advertiser needs to answer ("Is the advertiser buying or selling?", "(if selling) Where does the buyer send cash?", "(if buying) What's the max and min that you can send, or what amounts will you buy?  What shipping methods can you use?"

Another issue is how disputes will be resolved.  I feel that I am prepared to help two parties resolve a dispute in  a CBM trade, but I know that a scammer can always blame the carrier and we don't yet have any standard method of handling that.

Lastly, we don't need to protect users.  We can simply point out that "No one who worked on the code or took part in the discussions has been willing to guarantee that the CBM payment option is safe.  We all expect only the most seasoned and well behaved traders (and those who trade with them) to use the method with a high degree of success.  We encourage everyone else to try it with enough to pay for any lessons you learn along the way."  I believe this flies in the face of a lot of philosophies and outlooks on life, but I think a deep and honest look at how humans learn can resolve those differences.  Let's get back to trying stuff and learning!

**Comment by wiz** [comment]
With the USPMO payment method, we have the USPS to act as a trustworthy and neutral third party that can provide public verification for mediators / arbitrators in the event of a dispute in regards to the shipment being delivered and the money order cashed, not to mention a 20 year prison sentence for mail fraud, so the "chargeback risk" is mitigated.

With a "cash by mail" payment method, there is no way to resolve a trade dispute, since as a mediator I would have no way to know if a scammer didn't simply send an empty envelope to the seller.

**Comment by crocket** [comment]
@wiz That's why both sender and receiver have to film the entire shipping or unboxing process in one cut. If you film the entire shipping process in one cut and leave a unique mark inside the envelope, you are protected.

I recommend GoPro action cam mounted on a chest mount for filming the shipping or unboxing process in one cut.

**Comment by crocket** [comment]
@dscotese When you sell XMR for cash in a cash-by-mail deal, you have to specify a valid address. If you want to hide your home address, you need to acquire another address without revealing your identity. That's tricky.

**Comment by dscotese** [comment]
@crocket  I see your point.  It's definitely not a reason to avoid offering this payment method.

@wiz Are you able to decline mediating a trade with a particular payment method? Can't you decline to take on any particular mediating job?  For those who do have a way to do it, the only thing missing is the opportunity, and resolving this issue would fix that.

**Comment by wiz** [comment]
IMO the Mediators and Arbitrators don't really want to get in the situation where they need to make a judgement call based on who they think the scammer is based on evidence that can be manipulated, for one thing it creates a financial incentive for people to spend time crafting photoshopped or deep faked videos whatever you want to call it, but more importantly there should always be a clean cut way for anyone to verify what happened in a trade dispute, and if not, the payment method isn't secure and shouldn't be added to Bisq. All the secure fiat payment methods rely on a trusted third party like the post office or a bank. If it's cash in the mail there is no TTP so it's whoever can photoshop a video better to trick the arbitrator.

**Comment by ghost** [comment]
Maybe advanced users could choose to have the option to trade without support.  I've mentioned this idea [before](https://github.com/bisq-network/proposals/issues/234):
> Bisq should enable traders to negotiate the disputed payout between themselves. Since they are the sole owners of the 2-2 multisig they could do this anyway, but there exist no tools to let then collaborate in such a way.

Setting high security deposits should be enough incentive to complete the trade.  If they don't both sign the multisig payout then bisq gets a donation.  Knowing it is a risky option they would likely begin with a small trade amount.

**Comment by wiz** [comment]
> Maybe advanced users could choose to have the option to trade without support.

No, the game theory of a 2 of 2 multisig doesn't work without dispute resolution. For example if I am the buyer I can just refuse to make payment and refuse to sign any payout from the multisig that doesn't give 50% to me. Since you have more funds in the transaction than I do, you have no choice but to agree or lose 100% so this is a profitable ransom attack for the buyer. 

The only payment method that works like how you describe is Face to Face because it's a unique situation where both traders are physically in the same room, so traders can resolve their own disputes by calling the police or using local court system as the arbitrator or whatever they want.

**Comment by crocket** [comment]
For now, it's not possible to fake one-cut video evidence. Multi-cut video evidences can be manipulated.

If you film the entire shipping or unboxing process in one cut with an action cam, you are safe for now.

Without ready access to a really advanced artificial intelligence that can fake videos, you can't fake video evidences.
Such an AI doesn't exist, yet.

For now, the tricky part is acquiring an anonymous/pseudonomous proxy address for cash-by-mail deals.

**Comment by dmp1ce** [comment]
I agree with providing the trade option with NO moderator support. HIGH RISK warnings should be displayed for the trade.

Senders of the cash can mitigate the risk by knowing the address/recipient and sending in multiple smaller transactions. Receivers really have little transaction risk, but they do give information about their address and personal name.

I don't understand the ransom attack @wiz. Why can't the security deposit be equal for both sides? Isn't it only a profitable attack if the majority of sellers give into the ransom.

**Comment by crocket** [comment]
@dmp1ce Against surveillance teams who are looking to punish random people and discourage people from using cryptocurrency, revealing your address and your name is unwise. What if your government doesn't want you to obtain cryptocurrencies through cash-by-mail deals? They are already pressuring centralized exchanges into abandoning cash-by-mail deals.

If governments are determined to punish cash by mail deals by spending a lot of tax money, they can pull ransom attacks.

They can launch wars against unprotected cryptocurrency users. They understand wars cost a lot of money. They launched war on terrorism and war on drug. Why can they not launch war on money laundering or state that terrorists are using cash-by-mail deals to obtain cryptocurrencies anonymously? In this case, there is no CIA-funded terrorist boogeyman. People like us are painted as terrorists and money launderers. They will even create terrorist boogeymen on mainstream media if they need to. Someday, they will focus their firepower on cryptocurrency users.

They don't want money because they can print money out of nothing. They just need excuses to print more money. They want control over people. Surveillance is the key to control.

Censorship resistance is a tough goal. Poor people need protection from governments as well as from random fraudsters.

**Comment by crocket** [comment]
@mutagenfork I think your cash by mail protocol is a solid process. This should be the standard of cash-by-mail deals on bisq although those deals will have to be backed up by 2-of-3 multisig wallets.

All cash-by-mail dealers should mount action cams on their bodies by default.

Do we need anything else other than developer approval?

**Comment by crocket** [comment]
I realized that you can route cash through delivery forwarders. There are private companies that forward delivery boxes.
You have to pay for delivery fee twice and their handling fee which is cheap.
Sending cash by delivery is also an option because delivery is cheap in my country.

**Comment by pazza83** [comment]
Referred here via #4757 

Just trying to understand what would happen in the following scenarios as not everyone has access to a secure mail box or the same options for couriers:

- Bitcoin seller places offer to Sell 0.5 BTC on Bisq via 'cash in mail'
- Bitcoin buyer takes offer and sends the associated fiat amount for the BTC as per protocol documented above by @mutagenfork 
- Buyer sends fiat cash packaged in a tamper-resistant cash control bag via Courier
- Courier firm arrives at sellers address (house, apartment, business etc)
- Seller is not present at address so package is left at what the courier deems to be a secure / convenient location, or gets it signed by anyone else at the address eg receptionist
- Seller collects package completes trade

But what if the following happens:

1. Seller never gets package, or
2. Seller gets package from receptionist and pretends they never received it.
3. Seller gets package from convenient location porch, door step, shared mailbox, behind car etc and pretends they never received it.

In all these problem instances I would assume that the seller would be successful in mediation / arbitration and the buyer would have to take it up with the courier company and claim through the insurance offered if the courier firm could not provide proof of delivery to the seller or the sellers address?

Is it a case of Buyer Beware and for them to ensure they have adequate insurance to cover any losses?

Also with so many courier companies having different terms and conditions that would potentially affect the trading protocol. Eg FedEx does not allow cash to be sent, and specifically states it is not liable for any loss of cash. USPS, I am less familiar with, It appears that they offer a '[Registered Mail](https://faq.usps.com/s/article/What-is-Registered-Mail
)' service insured up to $50,000. This service appears to direct delivery only to the addressee (or addressee's authorized agent).  This would theoretically avoid the issues above. Not sure how they verify the person they are delivering to is the addressee? Just a signature? A signature seems less secure than an identify check. A scammer just deny it was them that signed for the package.

**Comment by crocket** [comment]
@pazza83 If the record says it was delivered, the arbitrator may just send bitcoin to the bitcoin buyer.
Also, people use cash-by-mail deals because they want privacy. If they value privacy, they wouldn't want any investigation on their activities. If a bitcoin seller pretended the cash did not arrive, the seller would get unwanted attention from agencies.

**Comment by pazza83** [comment]
@crocket thanks, that might put some seller's off using it.  

Where I live there is a good 5% when ordering something the courier will say it has been delivered when it hasn't! Maybe it is just my experience of receiving items from delivery drivers that are in a hurry to get to their next job and can't find anywhere suitable to park.  Packages can be marked as delivered when they have not been. 

If confirmed delivery on it's own is enough evidence to assess if the trade protocol was adhered to. What would stop an unscrupulous trader sending empty boxes to multiple peers in the hope they get marked as delivered and their counterparty does not have the package or has not taken video evidence of opening the package to prove otherwise.

The complication I see if that their are effectively 2 trade protocols in place.

Buyer to Seller through Bisq (both parties have equal control)
Buyer to Seller via Courier (buyer has the control of which courier to use and what insurance to purchase, seller has no control)

It would not be possible for the seller of BTC to be responsible for the buyer's choice of courier / insurance. It would also be hard to standardize this choice unless defining a particular country due to regional variations in couriers. 

As the buyer has control of both what courier service to use and how best to insure themselves I think the risk for ensuring package is delivered should be the BTC buyer's.

**Comment by dmp1ce** [comment]
The primary way Bisq could help Cash in Mail traders is by discovery. Cash in Mail deals are almost always based on trust, building up from small trades to start. I feel the same way about Face to Face local trades.

Just having a matching system for these Cash in Mail and Face to Face local trades would be very helpful IMHO. Just a glorified secure chatting service, which is capable of walking a trader through a risky trade, warning them of the potential hazards.

**Comment by crocket** [comment]
@pazza83 I think the person who has tighter video evidence should win....

**Comment by dscotese** [comment]
This discussion is available to anyone who wants to argue that there ought not to be a "Cash by Mail" option.  I imagine there are some people considering doing exactly that, and I haven't seen any good reason to agree with them.

It may be the case that users of BISQ expect BISQ development to include instructions for mediators that allow them to perform their duties __mindlessly__, but I submit that such users can be safely ignored.  I am willing to mediate in a cash by mail trade, and I may follow @crocket's suggestion of awarding whoever has better video evidence.  That's a good first strategy, but in my experience, a cheater usually outs himself very quickly without realizing it, so some actual mediation and consideration is due.

The payment method should be available already, and @mutagenfork's protocol seems the way to do it.  What's needed?  Someone to actually submit code?

**Comment by crocket** [comment]
> but in my experience, a cheater usually outs himself very quickly without realizing it

Where have you had experiences? What experiences?

**Comment by crocket** [comment]
I think floating prices should be implemented if they haven't been already. Localmonero supports floating prices because cash-by-mail deals happen over days. Floating prices protect you from high volatility.

**Comment by dscotese** [comment]
I was trading on LBC from 2012 till they stopped CBM ads, and have been trading CBM ever since on Paxful and Agoradesk.  I publish my number and am in several telegram groups, so I get many unsolicited messages and I'm constantly guiding the senders to reveal their level of honesty without realizing they are doing so.

Whether or not a CBM ad uses a floating price is a different issue from whether or not CBM ads should be available.  I agree that it's a good option.  If adding it slows down the availability of the method itself, I would like to see it postponed until the CBM method is available.

**Comment by crocket** [comment]
@dscotese Is it desirable for you to be contacted before a cash-by-mail deal begins?

**Comment by dscotese** [comment]
> @dscotese Is it desirable for you to be contacted before a cash-by-mail deal begins?

No but I do like when people contact me because my advertisement doesn't answer all their questions, so that I can update it.

Many new buyers want to contact me before they start a CBM trade, so from their perspective, the answer is yes.

**Comment by crocket** [comment]
@dscotese While I agree that being contacted before a deal is good, I don't like revealing my `cell` number or my telegram account. Doing so compromises privacy. A `cell` phone is literally designed to keep you in a digital cell. It's central to building digital prison.

**Comment by ghost** [comment]
As a user wanting to build a non-KYC stack, I prefer the CBM scenario over any method that involves banks or fintech.  It would be useful in countries that do not offer the USPSMO payment method.  I would avoid trading with a lot of different counterparties, preferring one a few reliable ones.  My strategy would be to start with a small trade and work higher as trust is built.  In short, I would use this payment method, in fact I already plan to use it on another platform if it is not provided on Bisq.

**Comment by pazza83** [comment]
Thanks for all the great feedback.

From your feedback it appears 'cash by mail' buyers would take additional precautions when buying by mail, mainly looking to establish a relationship with a regular seller.

> Just having a matching system for these Cash in Mail and Face to Face local trades would be very helpful IMHO. Just a glorified secure chatting service, which is capable of walking a trader through a risky trade, warning them of the potential hazards.

I like the @dmp1ce's idea Bisq being utalized for being a matching system. It seems reasonable to assume if both parties used Bisq to establish their trading relationship for their first trade they would continue to do so for future trades.

> It may be the case that users of BISQ expect BISQ development to include instructions for mediators that allow them to perform their duties mindlessly, but I submit that such users can be safely ignored. I am willing to mediate in a cash by mail trade, and I may follow @crocket's suggestion of awarding whoever has better video evidence. 

Thanks @dscotese for your comments. I have no experience of mediation other than being a trader on the other end of the process. I assume the mediators want to utilize payment methods that can be verified. Cash in mail brings additional complications to the verification process. Please can you advise how you would go about mediating the example I provided above: https://github.com/bisq-network/bisq/issues/1101#issuecomment-723430354

**Comment by pazza83** [comment]
> As a user wanting to build a non-KYC stack, I prefer the CBM scenario over any method that involves banks or fintech. It would be useful in countries that do not offer the USPSMO payment method. I would avoid trading with a lot of different counterparties, preferring one a few reliable ones. My strategy would be to start with a small trade and work higher as trust is built. In short, I would use this payment method, in fact I already plan to use it on another platform if it is not provided on Bisq.

@jmacxx This sounds sensible. Doing it this way would mean that you would decrease risks of any trades going wrong. If all users did this on Bisq CBM would be a great addition.

**Comment by pazza83** [comment]
> I was trading on LBC from 2012 till they stopped CBM ads, and have been trading CBM ever since on Paxful and Agoradesk. I publish my number and am in several telegram groups, so I get many unsolicited messages and I'm constantly guiding the senders to reveal their level of honesty without realizing they are doing so.

@dscotese thanks for sharing this. Sounds like you have built up relationships with people that want to use CBM.  In your experience was scammers utilizing CBM an issue with the other platforms? What made LBC stop traders advertising CBM offers?

**Comment by dscotese** [comment]
> ... Please can you advise how you would go about mediating the example I provided above: [#1101 (comment)](https://github.com/bisq-network/bisq/issues/1101#issuecomment-723430354)

I'll have to guess at a kind of worst-case scenario:
Buyer claims package delivered and seller isn't releasing. Buyer provides evidence in the form of tracking from the carrier upon my request or it's already in the trade.  I satisfy myself that a package was actually delivered.  Does seller agree and claim it was empty?  I'll assume yes.  I ask for video evidence from both, showing the contents of the package going into it (from buyer) and coming out (from seller).  If both videos look uncut and show a package otherwise undamaged*, I request both parties to submit affidavits stating what they did with the package.  I advise them that they can use these in any court they want to attempt to prove the other is lying.

If, during all this, both parties seem completely honest to me, I ask the buyer why he trusts that the carrier did not replace his package, asking him to carefully review the video evidence provided by the seller to see if the package looks like what he sent, even with details such as the writing on the package.  I suggest that the parties can work together to recover the funds from the carrier whose theorized theft has become the most reasonable explanation.

If I ultimately fail to sort it out, I leave the two parties with the two options: work together to recover the money from the carrier or work against each other to assign blame.

It's difficult for me to imagine it going this far.  It may end up that BISQ newbies get screwed in CBM operations more often because, as I can admit, there is a bias in me that long term users of BISQ have a reputation to uphold, and (I assume) disputes are noted so that if they regularly cheated newbies this way, they would soon have a crappy reputation if only for being "unlucky enough to have been used by a series of scammers for their CBM trades."

* Note that there are strategies not described here which I employ to tease out dishonesty.  They will eventually be well known and are already well known by those who trade regularly with me.  I do not share them publicly as that borders on training scammers to be be more effective.

> @dscotese thanks for sharing this. Sounds like you have built up relationships with people that want to use CBM. In your experience was scammers utilizing CBM an issue with the other platforms? What made LBC stop traders advertising CBM offers?

The main problem with scammers and CBM is that the scammer gets his victim to send a legitimate seller good money for a sale the scammer will never honor.  I make this difficult by requiring the mailpiece to include a note stating the username, platform, and reason for the cash.  LBC has no control over whether or not a seller does this, and no way to verify whether or not they do it, and I believe their reputation was suffering because of it, which is why they disallow it now.  I believe it has gotten harder to do as most people don't like such scamming and therefore share information about it so that awareness is ever more pervasive, making the scam more difficult.

One of my customers has analyzed the MITM scam extensively and points out that if the platform had a standard invoice created on behalf of the seller for the buyer to print and mail, it would be much more difficult for scammers to execute the MITM scam.  This approximates the note I require, but he's right that standardizing it a dn having the platform itself provide it would be even better.

**Comment by crocket** [comment]
> As a user wanting to build a non-KYC stack

@jmacxx What do you mean by a non-KYC stack? I know what KYC means. I just don't know what you mean by stack.

**Comment by ghost** [comment]
Its a term borrowed from the precious metals community, also used by some Bitcoiners.  Means "a large indefinite amount".

**Comment by crocket** [comment]
In software community, stack means a layer of softwares. I thought you were trying to build softwares.

**Comment by pazza83** [comment]
Hi @dmp1ce thanks for sharing all your useful info.

It all sounds good to me. If there is a strict protocol in place the risk of scammers can be mitigated. 

I think getting both parties to submit an affidavit should the mediator require it seems like a good solution for both parties and hopefully it would only be required where other forms of mediation have been exhausted.  I like the idea of both parties working together to resolve the issues. This should help to resolve most cases where the carrier is to blame,

Bisq traders do not have a reputation as such as to do so would be reduce privacy. However there are safeguards that can be put in place to ensure both new buyers and sellers start off with lower limits.

On my research of other P2P exchanges it appears that Local Bitcoins is the only one that does not support CBM. Paxful, Local Coin Swap, Local Cryptos, HodlHodl and Agoradesk all do.

I am in favour of Cash By Mail being added as a payment option in Bisq with a protocol as described above https://github.com/bisq-network/bisq/issues/1101#issuecomment-352630191 and utilizing a note including the Bisq 'Trade ID', and any message the Seller of BTC would like added.

It would be great to get the mediators @huey735 @bisq-knight @leo816 opinions on the concern raised @wiz about mediators and arbitrators not wanting to be in the situation where they need to make a judgement call based with reference to @dmp1ce comments https://github.com/bisq-network/bisq/issues/1101#issuecomment-724296022

Is there a consensus amongst the mediators about whether or not CMB should be added as a payment method to Bisq?

**Comment by chimp1984** [comment]
> For now, it's not possible to fake one-cut video evidence. Multi-cut video evidences can be manipulated.
> 
> If you film the entire shipping or unboxing process in one cut with an action cam, you are safe for now.
> 
> Without ready access to a really advanced artificial intelligence that can fake videos, you can't fake video evidences.
> Such an AI doesn't exist, yet.
> 
> For now, the tricky part is acquiring an anonymous/pseudonomous proxy address for cash-by-mail deals.

I have not read the full discussion so maybe I missed something important. 
But from my understanding it is super easy to fake such a video. I use a box with no money in or less as expected and open that and make the perfect non-cut video. How can it be proved that that box opened is the box from the peer? Address label, stamps all that can be easily photoshopped and glued to my manipulated box.

**Comment by chimp1984** [comment]
Another problem with the video is that users have different skill levels on doing that and it bring mediators in complicate situations if the video is not as clear as it should be but still somehow ok... It comes with a lot of blurry lines...

Another issue is fake money. I guess there mighe be scammer with pretty high quality falsifiactions which are hard to detect if one has not spent considerable time to become an expert on such stuff. 

I assume the platforms supporting CBM have additional security features like reputation. Bisq does not have that so I would see that as an invitation for scammers. Might work a while for small amounts but I don't see how that can be provided with some reasonable level of security.

What might work is it is combined with some other ideas, like https://github.com/bisq-network/proposals/issues/79 or BSQ bonds... Another option would be to user 100% security deposit or even higher and make clear that there is no dispute resolution like with F2F tades. But I guess if it gets that expensive nobody will use it anyway.    

We could delegate the reputation-check to the users by adding an email field in the offer so that the taker can contact the maker before the deal and they can exchange whatever they request like social media accounts. If there is enough trust provided by that this could reduce the risk but I would also argue that we should make clear that mediators and arbitrators cannot help if there is a dispute, so they have to be aware that funds can be kept forever in the multiSig or if one publishes the delayed payout tx its donated to Bisq without getting a reimbursement (as arbitrator cannot clearly find out who defaulted).

**Comment by crocket** [comment]
> But from my understanding it is super easy to fake such a video. I use a box with no money in or less as expected and open that and make the perfect non-cut video. How can it be proved that that box opened is the box from the peer? Address label, stamps all that can be easily photoshopped and glued to my manipulated box.

How is it easy? It's hard to imagine with your sketchy explanation. Can you demonstrate with a video? Fool us.
If I wanted to send cash in a delivery box, I would film myself inserting each bill one by one in the box.

As far as I know, there are AIs that can detect edited photos.

Counterfeit cash should not be considered until we solve the problem of fake videos.

**Comment by MwithM** [comment]
@crocket 
> If the record says it was delivered, the arbitrator may just send bitcoin to the bitcoin buyer. Also, people use cash-by-mail deals because they want privacy.

People could use CBM because they have found a way to scam other people.

@dscotese 
> The main problem with scammers and CBM is that the scammer gets his victim to send a legitimate seller good money for a sale the scammer will never honor. I make this difficult by requiring the mailpiece to include a note stating the username, platform, and reason for the cash. LBC has no control over whether or not a seller does this, and no way to verify whether or not they do it, and I believe their reputation was suffering because of it, which is why they disallow it now. I believe it has gotten harder to do as most people don't like such scamming and therefore share information about it so that awareness is ever more pervasive, making the scam more difficult.

Just as with Face to Face trades, I find it impossible for moderators and arbitrators to get tamper evidence of what happened during a trade. For example, a dishonest BTC seller could say that real money is counterfeit using a false bill verifier machine or a fake pen: video proof can't verify bills.
I don't think it's bad reputation what stopped Localbitcoins from using these payment methods, very similar platforms appeared the day after Localbitcoins ceased to do so. Reputation works well enough for this, people is used to it, although 100% security is impossible. They stopped this payment methods because of KYC/AML laws.

@chimp1984 
> Another option would be to user 100% security deposit or even higher and make clear that there is no dispute resolution like with F2F tades. But I guess if it gets that expensive nobody will use it anyway.

I think that the only way this payment methods  (face to face, gift cards, cash by mail) can work at Bisq is by [Mutual Assured Destruction](https://bisq.community/t/no-arbitrator-f2f-trades-with-multisig-2of2/7880), which would be really decentralized as there is no arbitration needed. 
The currrent max 50% security deposit can be enough for this, but it should be something that users choose and 100% might be an option. Very little changes, even not at all, need to be done to allow Bisq arbitrationless trades. We just need to acknowledge this possibility and let clear that traders should be willing to donate the security deposit and trading funds if something goes wrong, as that's the only way to protect from scammers.

**Comment by ghost** [comment]
I totally agree with what @MwithM said above (last paragraph).  I would like to see an option for MAD trades.  **Bisq is the only tool that allows two geographically separated people to easily create a multisig.**  It has the marketplace for advertising offers, and the trade protocol for managing the 2-of-2.  That's really powerful.  But it is under-utilized because of the mediation protocol which severely restricts what instruments can be traded.  MAD, anything can be traded.

**Comment by dscotese** [comment]
@jmacxx @MwithM 

MAD is an invitation to anyone who is upset to get revenge on anyone who advertises to the public.  It ought to be presented that way in order to protect people who don't realize that it can be used that way.

@chimp1984 

@crocket asks for a video, but you ought to give us two, one which shows you fooling us and one in which you don't fool us, and see if we can tell which is which.  I predict that most here will be able to tell, but that many will not be able to tell.  My experience, as I mentioned earlier, is that no one has tried it on me, and I've been doing CBM for a long time.  It's possible that one guy tried it on me, but he got an extra few percent of coin out of it, not an entire trade, which is why I think it wasn't a case of fooling me, but rather a case of me miscounting the cash I sent.

**Comment by ghost** [comment]
I don't understand why someone would want "revenge on anyone who advertises to the public".  Maybe if it was costless, but under MAD it costs the security deposit, and security deposits should be 5x or so more than the amount being traded.

For example: Alice, creates an offer to sell 0.01 BTC with 500% security deposit.  Bob takes offer & pays in the same amount of security deposit (0.05 BTC).  If Bob chooses to not honor the trade, he loses close to the same as Alice does (-0.05 vs Alice -0.06).  If Alice receives the cash and chooses not to honor the trade, its a huge net loss for her too (-0.05 vs Bob -0.05).  The key is deposits are larger than the trade amount.  They are incentivized to behave - you can't go round scamming people if you lose that much capital each time.

**Comment by ghost** [comment]
Well perhaps if it was a state agency they could burn money to attack people who advertise to the public.  But is that a realistic scenario?  They could more easily go after people who advertise Zelle or SEPA or any bank based trades and obviously that is not happening.

**Comment by chimp1984** [comment]
> > But from my understanding it is super easy to fake such a video. I use a box with no money in or less as expected and open that and make the perfect non-cut video. How can it be proved that that box opened is the box from the peer? Address label, stamps all that can be easily photoshopped and glued to my manipulated box.
> 
> How is it easy? It's hard to imagine with your sketchy explanation. Can you demonstrate with a video? Fool us.
> If I wanted to send cash in a delivery box, I would film myself inserting each bill one by one in the box.
> 
> As far as I know, there are AIs that can detect edited photos.
> 
> Counterfeit cash should not be considered until we solve the problem of fake videos.

I don't mean to edit the video, just to make video with the diff. box. Btw the same can be done from the senders side. The delivery service would be kind of the trusted third party, so to make it safe you would need to make a un-cut video from he sender where he packs the box and hand it over to the delivery company and get the tracking ID -all that in one video without cut and then the same for the receiver: video starts when he receives the box, verifies the tracking nr. opens all and counts all. 
I doubt that the filming of this whole process would be done properly by anyone. Also the delivery company or service man might not agree to be on video.

**Comment by chimp1984** [comment]
> a dishonest BTC seller could say that real money is counterfeit using a false bill verifier machine or a fake pen: video proof can't verify bills.

Good point!

**Comment by chimp1984** [comment]
> and I've been doing CBM for a long time

On which platforms and which security features do they offer? I assume reputation is the main feature they use, and that makes a big difference even its not safe as well to gives some game-theoretical ground (incentive for repeated games) which Bisq does not has.

**Comment by chimp1984** [comment]
> I don't understand why someone would want "revenge on anyone who advertises to the public". Maybe if it was costless, but under MAD it costs the security deposit, and security deposits should be 5x or so more than the amount being traded.
> 
> For example: Alice, creates an offer to sell 0.01 BTC with 500% security deposit. Bob takes offer & pays in the same amount of security deposit (0.05 BTC). If Bob chooses to not honor the trade, he loses close to the same as Alice does (-0.05 vs Alice -0.06). If Alice receives the cash and chooses not to honor the trade, its a huge net loss for her too (-0.05 vs Bob -0.05). The key is deposits are larger than the trade amount. They are incentivized to behave - you can't go round scamming people if you lose that much capital each time.

Yes I agree if MAD is used it has to be rather high. Also trade amounts should be rather low otherwise its more attractive to scammers (specially for using fake money and getting the btc). The blackmail risk has to be considered as well. There is always someoen who has more to lose, initially the seller, once the fait is transferred the buyer. So the other party could say lets make a new payout where i get half of the trade amount otherwise your (and mine) funds are locked forever or donated to Bisq. The higher the deposit in ration to the trade amount the lower the risk for that.

Another thing to consider it that we do not want payment methods which would create a lot of work for mediators. Those are costs for the DAO. So if a mediator need hours to resolve a CBM trade its a big loss for the DAO. Checking a video and getting into complicate disputes where there is no clear way how to find out the truth is a recipe for endless dispute cases.

So I would recommend to make it very clear that there is no dispute resolution like with F2F and delayed payout tx goes to Bisq as donation (no reimbursement). The delayed payout tx is a bit of protection against the blackmail risk as well, as the blackmailer does cannot just try it and wait until the peer maybe accept at some point (spec. if BTC price rise a lot it might become more reasonable ot accept). The peer could publish the del payout tx after time lock and then the blackmailer has lost his deposit.

**Comment by dscotese** [comment]
Security deposit for the win! I wasn't aware of that requirement.

**Comment by crocket** [comment]
I still think mediation can be cheap with cash by mail.

Entire shipping/unboxing process in one cut. Requiring specific cash verification methods.

Do you know any other method that doesn't involve government agencies as trusted third party?
I don't want to involve government agencies and banks.

What if I have no BTC and want to buy BTC through cash by mail?

**Comment by MwithM** [comment]
@crocket you can't use Bisq without BTC.

How do you solve the issue where I as a BTC seller say that your bills are fake? A video proof is not valid to verify that.

**Comment by crocket** [comment]
@MwithM I may ask the seller to send it back to me and pay for delivery. After receiving my cash, it's okay for me to cancel the deal.

**Comment by MwithM** [comment]
Giving back your fake money and cancelling the trade doesn't compensate me from the risk I suffered from keeping your fake money.

What if I send you fake money after I kept your real money?

**Comment by crocket** [comment]
Perhaps, I shall film the serial numbers on the bills? I don't think you can actually fake serial numbers and also the bills. Can you also fake holograms on bills? Learning basics of cash verification shouldn't be difficult.

According to my research, US intelligence agencies or north korean government faked US 100 dollar bills by using the same equipments that US government used to print the bills..... It's not something that small individuals can fake in a cost-efficient manner.

**Comment by chimp1984** [comment]
> Learning basics of cash verification shouldn't be difficult.

It is as our discussion shows. Experts might not get scammed but most traders will not be experts.

**Comment by chimp1984** [comment]
> It's not something that small individuals can fake in a cost-efficient manner.

Its a mafia business, and they will look for options to convert them to BTC.

**Comment by dscotese** [comment]
I have to agree with @crocket that it's easy enough to catch all the fakes that will get you in trouble.  We've probably all passed fake bills.  Some government agency is responsible for making sure the Fed is the only counterfeiter (oops, the only one printing money), I think the Secret Service.  If they need your assistance because someone pointed out that you had fake bills at some point, and you say "those bills went back to the sender," then you get to be a hero-martyr by going to jail (because you're required by law to turn the bills in and rat on your trading partner - which I NEVER DO and have NEVER been prosecuted for failing to do), or they just go back to the sender, hoping you're not lying.  It's hardly any risk.

@MwithM 
> Giving back your fake money and cancelling the trade doesn't compensate me from the risk I suffered from keeping your fake money.

I think that discovering I traded with someone trying to use fake money is far more compensation for "the risk I suffered from keeping your fake money" than I require. So I will sell with CBM and you will not. That's okay.

> What if I send you fake money after I kept your real money?

@crocket's idea about recording the serial numbers is a great idea.  There are often other marks and defects in bills that can be used to show the receiver returned something different from what the sender sent.  You are proposing foolish behaviors on the part of scammers.  May they engage in such and weed themselves out quickly!

I had assumed that BISQ traders have reputations to maintain.  BISQ may have much less value to me if it is too easy for someone with a bad reputation to escape it and continue using BISQ. I like the security deposit idea.  I did't know about the DAO as the source of compensation for mediators.  Thanks to everyone for contributing to this discussion because it has helped me learn a lot.  There are some people whom I've chosen to trust and we trade by mail.  Many people have chosen to trust me because I'm highly visible.  New contacts can currently find me ONLY through the sites where I advertise, which is where my interest in BISQ originates (as authorities get better at screwing up good things like they did with localbitcoins).  @dmp1ce's idea of contact-info-only "trades" is a great idea for that reason.

**Comment by pazza83** [comment]
This thread is now 3 years old! It would be great to reach a consensus if this payment method should be added or not by the end of 2020.

I am in favor of trying CBM. 

- F2F has similar risks to CBM. 
- The number of F2F transfers on Bisq are very low.  
- I am not aware of any major incidences with F2F trades that have occurred on Bisq
- Their is a strong use case for CBM and their buyers and sellers that would like to utilize this payment method
- CBM would increase trading opportunities and make markets more accessible for traders with privacy concerns or with poor access to banking and financial institutions
- I think that trading in cash for BTC is one of Bisq's main reasons for exsisting. Bitcoin Square > BitSquare > Bisq. CBM is just an easy way for people with cash to access the square :)

I would suggest CBM as a payment option is implemented and kept under review, and if necessary the protocol developed  to be as secure as possible.

I think the risk is higher, but people choosing to trade this way will almost certainly be aware of the additional risks. They are happy with these risk for the increased privacy that CBM brings. Bisq trades should be expected a certain level of caution and takes steps to protect themselves when completing CDM trades, as would be expected with F2F trades or any other bank transaction.

I think the potential upsides this payment method brings outweighs the risks of its potential downsides. For this reason I would like to see it added.

If their is disagreement about wither CBM should be tried as a payment option the issue should be put to a vote by the DAO.  

It would be great to get the mediators @huey735 @Bisq-knight @leo816 opinions. As they would have to be happy to potential mediate any trades that require their input.

**Comment by chimp1984** [comment]
I am in favor of trying it out as well. But I guess a higher security deposit it the only option to make it a bit more safe and we should make clear that mediators are unable to make a judgement. They still could try to help, but both traders need to be aware that in conflict situations there is no "judge" and funds stayed locked up forever or are donated to the DAO in case one publishes the delayed payout tx.

If the higher security deposit is hard enforced or we just show a popup with telling the users the risks it up for debate. As well as the hight of it. I fear a 100% deposit will end up that nobody will use it.

**Comment by pazza83** [comment]
> I am in favor of trying it out as well. But I guess a higher security deposit it the only option to make it a bit more safe and we should make clear that mediators are unable to make a judgement. They still could try to help, but both traders need to be aware that in conflict situations there is no "judge" and funds stayed locked up forever or are donated to the DAO in case one publishes the delayed payout tx.

I think in a low percentage of cases they will not be able to make a judgement.  In the majority of cases I am sure it will be clear who they would side with based on the evidence presented.  No need for funds to stay locked up forever. They can just be sent to the arbitrator where one side does not accept the mediator's proposed solution. 

In the rare cases where their the mediator cannot make judgement then the funds can go to the arbitrator. The funds can then be burned by burning man. Both parties can then submit a proposal for reimbursement from the DAO in BSQ by and provide their evidence. Unless new evidence was presented at this stage it is unlikely either party would get their funds back.

It can be handled the same way F2F trades currently are. I do not think their is much more risk, but just more volume, so more potential for CBM trades to end in mediation than F2F trades.

**Comment by chimp1984** [comment]
> Both parties can then submit a proposal for reimbursement from the DAO in BSQ by and provide their evidence. Unless new evidence was presented at this stage it is unlikely either party would get their funds back.

Can you make a suggestion for a text for the rules in a popup we need to display the traders? It should make it clear that as there is no way to find clear evidence for mediators / arbitrators traders have to accept that there might be no resolution in dispute and no reimbursement. We can keep the option open but we should make very clear the different situation compared to other payment methods where usually a tamperproof evidence is at least in theory possible (PageSigner) and in most cases a practical sufficiently working way to proof exist (e.g. show video of mobile app or banking webpage to provide evidence to support agent).

I think the discussed options for providing "evidence" should not be used as they are just "snake oil" as there are too many ways to scam those. It would just create a wrong impression for security where the more sophisticated scammer will find more naive victims.

We also should not get tricked by the fact that there are very few real disputes in Bisq. That is a result that we are very restrictive with payment methods and have the option to get tamper proof evidence. If that is not in place scammers might find Bisq more attractive and we saw it with Venmo and CashApp that they quickly came then when they see an opportunity.

**Comment by crocket** [comment]
@chimp1984 You are suggesting that we give up arbitration even before trying to arbitrate cash-by-mail deals because you theorize that it is not possible to arbitrate cash-by-mail deals. I don't like giving up before even trying. Localmonero has arbitrated cash-by-mail deals. We can hear from users and arbitrators on localmonero and agoradesk.

I heard enough theory. We need to hear direct experiences.

**Comment by MwithM** [comment]
The only reason I've made F2F payments at Bisq is because settlement needs to be done in person, so at least you would already know if you've been scammed and who did that. Even though, I always preferred MAD if it were a possibility, because in case of dispute, arbitrators can't know what happened during a trade and also, it would make unnecessary to bring the laptop to a trade setlement and seller could count and verify bills safely at home.
With CBM there's too many things that can go wrong. I'm quite sure that the main way to arbitrate this at other places has been using reputation, and Bisq doesn't work that way.
If arbitrator agrees, maybe the best way is to leave users choose if they prefer to use arbitration system or not when making an offer so before starting the trade it's known if arbitration will be available or not. Another point is that if mediation is a possibility, and whether that's positive or not. I'm inclined to think that if users agree to a payout it should be ok for Bisq, but it also leaves the window open for ransom.

About recommended security deposit, I would personally not use more than 100%, and I think that a min of 50% could be ok. Now when taking the offer it's clear what security deposit % is set so users should be able to take responsible choices.

**Comment by pazza83** [comment]
@chimp1984 

> Can you make a suggestion for a text for the rules in a popup we need to display the traders?

Yes, no problems at what point is this shown to the traders? Is there something similar for F2F trades currently?

**Comment by pazza83** [comment]
> The only reason I've made F2F payments at Bisq is because settlement needs to be done in person, so at least you would already know if you've been scammed and who did that. 
> With CBM there's too many things that can go wrong. I'm quite sure that the main way to arbitrate this at other places has been using reputation, and Bisq doesn't work that way.

I think this risk for F2F vs CBM is different for different users. Maybe a young female would feel less at risk as using CBM as a payment option than F2F. 



I do not think F2F is inherently less risky than CBM. It just depends on ones perspectives.
 
> If arbitrator agrees, maybe the best way is to leave users choose if they prefer to use arbitration system or not when making an offer so before starting the trade it's known if arbitration will be available or not. Another point is that if mediation is a possibility, and whether that's positive or not. I'm inclined to think that if users agree to a payout it should be ok for Bisq, but it also leaves the window open for ransom.

I think mediation followed by arbitration for CBM trades would be suitable until such time another solution is possible or required. Most trades end up resolved through mediation with relatively few going to the arbitrators. I think CBM would be likewise.

> About recommended security deposit, I would personally not use more than 100%, and I think that a min of 50% could be ok. Now when taking the offer it's clear what security deposit % is set so users should be able to take responsible choices.

Yes i think sellers might want buyers to place a larger % for these trades. Can it not just be left to those making the offers. When making an offer if someone want to buy/sell BTC with a 15% security deposit. Why not let them?

**Comment by chimp1984** [comment]
> @chimp1984
> 
> > Can you make a suggestion for a text for the rules in a popup we need to display the traders?
> 
> Yes, no problems at what point is this shown to the traders? Is there something similar for F2F trades currently?

Yes F2F is a good example. There are several. But main popup is after account creation. The optional others are at trade start and payment sent. Its more of reminders as ppl might forget details between account creation and trade.

**Comment by chimp1984** [comment]
> I think this risk for F2F vs CBM is different for different users. Maybe a young female would feel less at risk as using CBM as a payment option than F2F.
> 
> I do not think F2F is inherently less risky than CBM. It just depends on ones perspectives.

I think it is different from the potential resolution chances:
With F2F if done right you meet at some public place where some cameras or witnesses might be of use in case you get scammed. So a pro scammer has it harder here to get 100% success.

With CBM a pro scammer (e.g. very high quality fake notes) will get a very high success rate and you cannot do anything to catch him and mediation/arbitration will not be of help (no way the mediator can verify that notes have been fake). 

But they also have different worst case risks: With F2f you can risk your life in the worst case, with CMB you cannot lose more than money. 

One aspect to not to underestimate regarding risk is the fact that the buyer knows the home address of the seller. So that makes it actually for the seller similar risky as F2F.

**Comment by chimp1984** [comment]
> Yes i think sellers might want buyers to place a larger % for these trades. Can it not just be left to those making the offers. When making an offer if someone want to buy/sell BTC with a 15% security deposit. Why not let them?

Yes I would also recommend that, but with diff. min/max values. Currently is 15-50%. Maybe 30-100% would be more appropriate?

**Comment by pazza83** [comment]
> > @chimp1984
> > > Can you make a suggestion for a text for the rules in a popup we need to display the traders?
> > 
> > 
> > Yes, no problems at what point is this shown to the traders? Is there something similar for F2F trades currently?
> 
> Yes F2F is a good example. There are several. But main popup is after account creation. The optional others are at trade start and payment sent. Its more of reminders as ppl might forget details between account creation and trade.

Thanks, I will take a look.

I notice F2F trades also have information that the maker can determine that can displayed for the taker to see before the trade. Would be good if this was possible for CBM also. It would allow maker to stipulate their terms as per protocol

**Comment by chimp1984** [comment]
> I notice F2F trades also have information that the maker can determine that can displayed for the taker to see before the trade. Would be good if this was possible for CBM also. It would allow maker to stipulate their terms as per protocol

Agree

**Comment by pazza83** [comment]
> I think it is different from the potential resolution chances:
> With F2F if done right you meet at some public place where some cameras or witnesses might be of use in case you get scammed. So a pro scammer has it harder here to get 100% success.

Yes but that is comparing F2F done right with CBM done wrong.

> With CBM a pro scammer (e.g. very high quality fake notes) will get a very high success rate and you cannot do anything to catch him and mediation/arbitration will not be of help (no way the mediator can verify that notes have been fake).

Same for F2F scammer with high quality fake notes. 
 
> But they also have different worst case risks: With F2f you can risk your life in the worst case, with CMB you cannot lose more than money.

Agreed, and a good point !

> One aspect to not to underestimate regarding risk is the fact that the buyer knows the home address of the seller. So that makes it actually for the seller similar risky as F2F.

Only if they use their home address. People can receive deliveries to multiple addresses. (post office, parcel collection points etc).

**Comment by MwithM** [comment]
Not using your home address is almost a must for CBM.

> I think mediation followed by arbitration for CBM trades would be suitable until such time another solution is possible or required. Most trades end up resolved through mediation with relatively few going to the arbitrators. I think CBM would be likewise.

I still think we need mediator's and @RefundAgent 's perspective.

**Comment by crocket** [comment]
> Not using your home address is almost a must for CBM.

How do you avoid using your home address?

**Comment by MwithM** [comment]
It depends on the services available for each country and receiver's imagination.
One option for Spain is to use a postal box. Work address can be used as well.

**Comment by pazza83** [comment]
I have made some proposals for putting a system in place to speed up decisions around new payment methods being added. Would welcome any comments https://github.com/bisq-network/growth/issues/206

**Comment by pazza83** [comment]
Hi @bisq-network/mediators and @bisq-network/arbitrators I would like to propose implementing CBM as a new payment method. Please let me know if you any feedback or concerns. There is some discussion above with concerns about how CMB can be mediated / arbitrated.

**Comment by dave2014btc** [comment]
Wow a lot of comments now, to start off im talking from experience sending cash in the mail in uk and e.u, as i said before the chances of something going wrong are under 0.5% based on my own experience. 
To begin with I count money under camera and run it through my uv light, i pack it seal and stick label on. all this is recorded. 
i then send it out but as a terms of service the buyer accepts all risks associated with sending e.g. lost stolen replaced and so on. 
what incentive do i have to not switch out the cash or whatever ? 
my feedback if i do one bad trade it goes from 100% positive to 99% which will give doubt to future customers and cost me much more down the line.
Also within the uk we are able to insure up to £7500 in the post so if lost or stolen we can claim and do a criminal case. 
To be clear the buyer accepts all risks and they fully know that when opening trade with me so the only reason i would have a moderator dispute against me was if i never decided to send it.

we are making this far more complicated than it needs to be. it could be a simple as warn people before opening trade for cash by mail along the lines of " opening a cash by mail trade is risky as once it is sent the buyer accepts responsibility" my customers will still buy and you have covered your side. you could always ask the person opening it to record when opening but in my situation it dose not matter as once i have sent i have done my part. obviously if the cash was missing i would help with claiming from royal mail but the btc would be released regardless as the risk is not with the trader. with e.u packages you cannot insure but never had one go missing yet. 
maybe start with only letting people with say 50 or 100 feedback be able to advertise on cash by mail so they have feedback to loose if anything gose wrong. btw i haven't used bisq yet so not sure how feedback works, i use pax, localcryptos and localcoinswap.

if anyone could maybe improve on my ideas that would be good. 
Thanks

**Comment by crocket** [comment]
@dave2014btc How did cash by mail deals go wrong in your experiences?

**Comment by pazza83** [comment]
Hi @dave2014btc 

Thanks for all the useful info.

One of the challenges with Bisq is that their is no feedback system. This is to protect the privacy of all traders.

When a trade occurs on Bisq you will only be able to tell if you have traded with the person before or not. if so it will let you know how many trades you have completed with them.

Would you trade if buyer was to accept all responsibility until cash delivery occurred? The insurance you mention would protect the buyer in case of problems with the courier but it would not protect the seller.

**Comment by ghost** [comment]
@crocket you said we need to hear direct experiences, @dave2014btc gave his experience and it hinged on the reputation system.  There is no way for a user to build reputation on Bisq.  IMHO, CBM won't work for the current trading & mediation model.  I think we should not add it at this time.  [MAD](https://bisq.community/t/no-arbitrator-f2f-trades-with-multisig-2of2/7880) trades could be discussed in a separate proposal.

There have been lots of good ideas tossed around in this issue, some people want the protection of arbitration, others don't & prefer MAD.  Some people (Bisq OG's) have warned that it is impossible to arbitrate this payment method.  I don't see any consensus.  Perhaps it needs to go to a DAO vote?

**Comment by pazza83** [comment]
I am happy with a DAO vote. There has been ample time for everyone to give an opinion.  Having a definitive vote either way would be good. I will propose one for this cycle.

**Comment by dave2014btc** [comment]
@Jmacxx it hinges on regular customers coming back and trusting me but also the fact that the sender takes responsibility. the feedback just helps get more customers or less depending on what it is. im not looking for find new customers anyway but just have a decentralised place to send my old ones. saying i have been trading for over 5 years with almost no problems with the cash by mail side (i had far more problems with charge backs thats why i don't use bank transfer) yet you would still use bank transfer when money can be sent back? in uk we have royal mail, we can insure the cash and we can open a criminal case if someone stole the money or never sent (basically fraud). with cash in mail they is a end point its not anonymous so if someone dose try and scam then you can wait on the dispute till the police have had there say. ? regardless of that i just want somewhere i can direct my customers where they feel its a safe escrow. if it helps you can always put a warning when doing them trades letting people know its heightened risk. in the uk for example cash in person is far more dangerous due to gang thefts and so on. i open all packages under camera as well which could be a requirement even if someone sent 50k its still not worth it financially to try and edit a video into a fake. are they any other cash by mail traders from other countries as i do understand united kingdom is in a very easy situation to do cash by mail due to the insurance royal mail offer and the fact they will insure cash as i know most country's mail services won't. maybe just make it available for u.k ? hopefully we can sort something as i love the idea of bisq. good luck on the voting.   my biggest problems to the people asking before is when people say they have sent but haven't or people opening trade but waiting till price moves up before sending but them people are 1 in 1000. also no one can be bothered to try and scam with there address visible due to risk of police turning up. are they any other cash by mail traders from other country's reading who can give there perspective on this.

**Comment by pazza83** [comment]
Please see below a summary of the proposal to add Cash by Mail as a payment option to Bisq.

Cash by Mail has many desirable features for use as a payment method.

Some of the contentious issues are around protentional concerns with what evidence can be provided for delivery / receipt of the cash, and if the cash is or is not genuine.  

There are also concerns around if potential mediation / arbitration of Cash by Mail trades can be facilitated by the current mediation and arbitration systems in place. 

I am in favor of adding Cash by Mail as a payment option and keeping it under close review for the next 12 months.  At 12 months the benefits/negatives will be more obvious and a decision can be made to continue or retire this payment option. 

For the purpose of the DAO vote I propose using the below parameters. These can be further agreed if needed. 

Please see the vote as being to vote in agreement or disagreement with the principle that "Cash by mail" should be added as a payment option to Bisq. 

**Trade sizes**: Default as per https://github.com/bisq-network/proposals/issues/264
**Signing required**: No
**Deposit amount**: 15-50% 
**Maker / Taker fees** preferred: 1% maker, 1% taker (to compensate for mediation complexities)
**Maker / Taker fees** minimum: Default
**Trade protocol**: Clearly defined both in terms of Bisq's minimum expectations, and also by the seller to add any personal / localization information (this will be visible before trade accepted on Bisq in the same way as F2F traders can provide additional information). For reference see https://github.com/bisq-network/bisq/issues/1101#issuecomment-352630191
**Risk of sending cash**: The buyer is responsible for ensuring the seller takes delivery of the cash. They can choose an appropriate courier and insurance to protect themselves from any problems that occur during delivery.
**Warnings**: Clearly defined at all user interaction points; wiki, within client etc
**Review**: all the above parameters to be reviewed after 12 months of inclusion. Review to include the possibility of payment method to be withdrawn.

  
Please see below a summary of the features of Cash by Mail:


| Essential   | Desirable | Definite No’s |
| ------------------- | ------------------- | ------------------- |
| ~~Very low risk of chargeback~~  | No risk of chargeback  | ~~< very low risk of chargeback~~  |
|  ~~Way to verify the sender in the received payment~~  | Ability to enter Trade ID as reference  | ~~No way to verify the sender in the received payment~~  |
| Trade time less than one week  | ~~Instant payment~~  | ~~Trade time more than one week~~  |
| Singular Fiat currency per account, traders may wish to create multiple accounts  | ~~Multi-currency~~  | ~~Not a payment method for fiat currency~~  |
| Significant user base  |  ~~Large user base~~  | ~~No significant user base~~  |
| High usability  | ~~High usability and great user experience~~  | ~~< high amount usability~~  |
| No KYC required | Allows users to trade with upmost privacy. Minimal identifying information as possible (no names, email, phone etc required) | ~~Some KYC required (proof of address, ID, selfie)~~  |
| ~~Low risk of scam attempts~~  | ~~Trades are as safe and secure for as many users as possible~~  | ???< low risk of scam attempts |
| ???Traders can provide evidence of payment / receipt  | ~~Verification of payment can be made using PageSigner or similar~~ | ???Traders will be unable to provide evidence of payment / receipt  |
| ~~Minimum limit at least equal to at least account limits protocols~~  |  No minimum limits  | ~~Minimum limit not able to achieve account limits protocols~~  |
| ~~Maximum limits equal to at least 0.01 BTC~~ | Large payment limits up to 2 BTC  | ~~Maximum limit is less than 0.01 BTC~~  |
| Likely to increase liquidity  | ~~Likely to open markets for different countries and currencies~~  | ~~Likely to decrease liquidity~~  |
| ???Low risk of mediation  | ~~Very low risk of mediation~~  | ???< low risk of mediation  |
| Low risk for traders from government agencies  | ~~No risk for traders from government agencies~~ | ~~< low risk for traders from government agencies~~  |
| Fees are not be a barrier to trading  | ~~No fees for transactions~~  | ~~Fees are a barrier to trading~~  |
| Only minor changes needed to trade protocol  | ~~No changes needed to trade protocol~~  | ~~> Minor changes needed to trade protocol~~  |

**Comment by MwithM** [comment]
Maker and taker fees should not be higher than other payment methods. Dispute costs should not be subsidized by all the users but paid by those who use mediation and arbitration.

I think this payment method could be multicurrency, as most postal cards can cross borders.

**Comment by chimp1984** [comment]
>Deposit amount: 30-100%

Just want to mention that this is a bit extra dev effort and we usually dont have payment method specific deposits.
I think if MAD is a main protection feature we though have to do that.

>Maker / Taker fees: 1% maker, 1% taker (to compensate for mediation complexities)

Same here. Will be extra dev effort and might conflict or complicate possible future changes as https://github.com/bisq-network/proposals/issues/280.
I think the extra dev effort does not justify the potential extra revenue (will depend how much it will be used). We could do it later as well if we think its justified.

**Comment by chimp1984** [comment]
> I think this payment method could be multicurrency, as most postal cards can cross borders.

Not sure of the insured methods work cross border, I doubt.

**Comment by MwithM** [comment]
> > I think this payment method could be multicurrency, as most postal cards can cross borders.
> 
> Not sure of the insured methods work cross border, I doubt.

Is to be insured a requirement?
If it's kind of common to send packages  with different currencies crossing borders, changing this will be easy? If it isn't, i think it's best to let traders decide if they wish to use it. I was going to oppose to not having a smaller max trading amount, but I followed the same reasoning: traders should know the risks they take and act accordingly.

**Comment by pazza83** [comment]
Yes :) Think there will be discussion around the protocols, and that can take into account dev time. I will update it to be same as normal.

Yes I agree in some instances it can be multicurrency.

**Comment by MwithM** [comment]
@jmacxx If mediation and arbitrator teams want to go ahead, there's not much more to say through a DAO vote.
I know I won't use this payment method with arbitration but I won't vote against that DAO proposal. If there's traders willing to use it this way they just need mediators to work on it.

**Comment by pazza83** [comment]
I don't think any current mediators have opposed it.

**Comment by chimp1984** [comment]
@dave2014btc 
Thanks for your inputs!
As others mentioned already Bisq does not has a centralized reputation system as that would not work with protection of privacy. As the main security tool on other platfroms is that reputation and as you stated thats a main motivation to not scam as you want to be in the game for longer time we need other protection.
Do you think that a high security deposit (15% is min. anyway for all methods) will be accepted by your trade peers? Discussed amounts are 50-100% deposit for both sides which gets refunded once trade is completed. The deposit is in the 2of2 multisig, no trusted third party. So full control by the 2 traders. But that opens also a new risk of blackmail. There is always one trader who has more to lose at a certain point in the protocol. The one who has less to lose could then blackmail the other. For smaller amounts and for high deposits that risk become lower but it cannot be eliminated.

Regarding privacy/physical security:
Are most users using their home address? If so a market maker would collect a lot of sensible and security relevant data. E.g. if you have collected 100s or 1000s of addresses of BTC holders and you get hacked your computer the hacker know where BTC holders life. Just imagine BTC goes to 100k - that will be a very real risk exposure to those traders to get robbed at their home.
Are people on other platforms not concerned about that? Or do they use mailboxes instead of home addresses? 
Another risk is that regulators (market makers acting as high volume pro traders might easily become a target of regulatory requirements) capture such a market maker and confiscate that data and know immediately a large chunk of the BTC holders. Atm of course its not illegal to hold/trade BTC but times can change and some ugly tendencies are on the way to become law in the EU (enforcement to register all your btc addresses). Or just unclear tax situation where the citizen is guilty by default and need to proof that he is innocent (as it is the default tax authority attitute in Spain or Italy). That could be used very easily for criminalisation of BTC holders at any time and having honeypots where they get a large chunk of user data is a serious risk.

Regarding video/proof:
My main concern is not fake video which is indeed hard to do. But the process of puting in the money until handing over to the delivery company would need to be filmed without any cut and that is not feasible. Same for receiver, to get a real proof it would require a video showing the point of receipt from delivery company until unpacking and counting.
And then there is the case with fake money. If high quality fake it is likely hard to detect for non professionals with more advanced equipment. And its basically impossible to proof on video. So buyer could always claim he got problems later when using the cash at a bank and there is no possible proof for that.
All that ends up that the mediators cannot make a judegement in real disputes as they lack of a solid proof. They still can help with smaller problems not related to scam attempts, but the security of all other payment methods in Bisq (excl. F2F) is based on the fact that there are ways to get a solid proof so scammers will not even try is as they know they would lose.

**Comment by chimp1984** [comment]
@wiz @huey735 @Bisq-knight @leo816  Could you add your view?

**Comment by chimp1984** [comment]
> traders should know the risks

Please consider that scams will not only hurt the trader but also have impact on Bisq:
- Reputation damage if lots of scams happen
- Higher effort for mediators/arbitrators/DAO stakeholders if it goes for reimbursement requests
- Effort for all contributors to deal with that (e.g. looking for improvements - this discussion already genereates a lot of effort)
- If Bisq is used as exit platform for scammers it attracts LE and can create further problems for Bisq.

**Comment by chimp1984** [comment]
> DAO vote

I would prefer to use a DAO vote only as a last resort if there are 2 camps cleary pro and contra. But I don't see that.
I am for example in favor to add it but in a cautious way and not with naive assumptions which can easily lead to attract scammers or cause quickly further adoption or removal.

My suggestion (most are in sync with @pazza83's summary):
- Use high sec deposit  (50-100%)
- Maker very clear that mediation can not guarantee to be able to help in case of fraud as there is no way for getting a solid proof. Best effort still can be done but traders have to know in advance higher risk of forever locked up funds or that the funds are donated to Bisq without reimbursement (in case one trader publishes the delayed payout tx). 
- I would suggest that mediator decision is final. If the mediator is unable to come to a clear conclusion the refund agent and the DAO will not do a reimbursement. Otherwise they follow mediators payout suggestion.
- Suggest users to use insured delivery
- Use free text form as in F2F for details and TOC of maker
- Trade limits 0.25 BTC, no account signing

**Comment by pazza83** [comment]
> I would prefer to use a DAO vote only as a last resort

Can you let me know thoughts behind why only as a last resort? 

Can a vote not be used so speed up a process to gain agreement for or against a proposal quickly?

**Comment by ghost** [comment]
@pazza83 `[...] as a mediator I would have no way to know if a scammer didn't simply send an empty envelope to the seller.` https://github.com/bisq-network/bisq/issues/1101#issuecomment-702456502 by @wiz

**Comment by pazza83** [comment]
I do not think @Wiz is currently a mediator

**Comment by chimp1984** [comment]
> I do not think @wiz is currently a mediator

He revoked but then later become again afaik. Not 100% sure though.

**Comment by chimp1984** [comment]
> Can you let me know thoughts behind why only as a last resort?

I think voting got such a good reputation as our modern democracy has forgotten about the original idea where democracy was about public discourse and finding consensus. Only in the event that no consensus can be found but all arguments have been laid out and are understood by the partizipants voting was the last resort to come to a result. Voting is dictatorship of the majority over the minority and creates a winner and a loser group, so it has potentially negative impact to a group (cause devision). In contrast to find a consensus makes the group stronger as they feel more aligned with the result they could create together. Also the result is then supported by the whole group and not only by the winner group (losers tend to form opposition). I don't want to over-interpret that for our small example here, but those are the reasons why I prefer to avoid DAO voting if not needed.

It also reduces outcomes to binary outcomes which might not always reflect good the situation. And it adds delay with the DAO cycle...

> Can a vote not be used so speed up a process to gain agreement for or against a proposal quickly?

Sure if things get stuck it is for sure a tool to be used. I just don't have the feeling that this proposal suffers on stale mate problems, but rather of different expertise and understanding of risks (CBM experts vs. Bisq experts -> both tend to lack understanding of the other domain). 
I know that issue is very old, but I think there have never been a serious discussion in the past, so it really started to me only recently...

**Comment by pazza83** [comment]
Thanks for letting me know that sounds great.

Does anyone have any objections to @chimp1984's proposal to proceed in a cautious way that reduces the risk of attracting scammers as per https://github.com/bisq-network/bisq/issues/1101#issuecomment-731279539

It would also be great to hear from @jmacxx and @wiz to hear your feedback on @chimp1984's proposal with reference to your expressed concerns.

It would also be great to hear from @huey735, @huey735, @Bisq-knight, and @leo816 opinions on the concern raised @wiz about mediators and arbitrators not wanting to be in the situation where they need to make a judgement call based with reference to @dmp1ce https://github.com/bisq-network/bisq/issues/1101#issuecomment-724296022

🥇 Fun Fact: This issue is the most commented ever on https://github.com/bisq-network/bisq/issues

**Comment by chimp1984** [comment]
The many comments are promising that this payment method might gain traction!

**Comment by crocket** [comment]
I prefer lazy consensus after getting work done to concensus before starting work. Faster iterations enabled by lazy consensus make an organization a fast learner. Learning from direct experiences is always better than paralysis by thought experiment committee.

Merge a pull request first. Seek forgiveness later. Review after merging. Lazy consensus also attracts more free labor because people like projects that merge fast.

My small projects practice lazy consensus.

Add cash by mail. If it turns out to be a pain in the ass, you can scrap it later, or you can improve it incrementally.
Learning from direct experiences...

**Comment by pazza83** [comment]
I am keen to come to a decision on this by the end of this cycle.

I am proposing the add CBM as a payment option to Bisq. This seems to be roughly in line with the majority consensus. 

It would be great to have @bisq-network/mediators input @huey735, @huey735, @Bisq-knight, @wiz and @leo816 please let me know if you have any concerns regarding mediating CBM trades.

**Comment by pazza83** [comment]
As per [proposal for existing payment method editing process and estimated timelines](https://github.com/bisq-network/growth/issues/207)

I am recommending this payment method for addition.  

No concerns have been raised from @bisq-network/mediators so would like to make plans for getting this added in the next cycle. 

@huey735, @huey735, @Bisq-knight, @wiz and @leo816 if you do have any concerns share them on this issue before the end of the cycle 27th Dec 2020.

**Comment by pazza83** [comment]
Thanks to @jmacxx who has started working to implement this 👍

**Comment by pazza83** [comment]
Hi the first draft of the [Cash by Mail Wiki for Bisq is online](https://bisq.wiki/Cash_by_Mail)

Please share your thoughts.

I am wondering if phone number should also be included at the payment account creation stage so that this can be passed on the the courier. Would Sellers of BTC using cash by mail be happy to share their phone number with the buyer / courier company to facilitate a smooth delivery?

**Comment by crocket** [comment]
Phone numbers can be included, but you shouldn't enforce or validate it.

**Comment by pazza83** [comment]
I think then best to leave for traders to communicate in trader chat. Numbers might change also depending on preferences at the time of trading.

**Comment by pazza83** [comment]
Cash by mail is now possible with Bisq https://bisq.wiki/Cash_by_mail
